great-idea/css/index.css

```diff
@@ -63,4 +65,96 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	display: flex;
+	flex-flow: row wrap;
+	padding: 2% 5% 5% 5%;
+	margin: 0 10%;
+	background: #fff;
+}
+
+header nav ul li{ display: inline; justify-content: space-between; }
```

Comment:
Fantastic job on converting the project using flexbox, could you install `prettier` as vscode plugin for code readability